<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WAnticamera</class>
 <widget class="QWidget" name="WAnticamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>905</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="bagnoBtn">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>150</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Bagno</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ripostiglioBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>320</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ripostiglio</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cameraBtn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Camera da Letto</string>
   </property>
  </widget>
  <widget class="QPushButton" name="openSpaceBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>350</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>OpenSpace</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
